CodingStyle.cc

```diff
@@ -0,0 +1,112 @@
+/*=====================================================================
+ 
+ QGroundControl Open Source Ground Control Station
+ 
+ (c) 2009 - 2015 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ 
```

Comment:
I usually add the header if it is missing to a file I'm working in. Sometime's I'll update the copyright, but mostly I forget.